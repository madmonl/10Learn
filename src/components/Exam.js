import React, { Fragment, Component } from 'react';
import ExamUpperNavigation from './ExamUpperNavigation';
import ExamBody from './ExamBody';
import ExamLowerNavigation from './ExamLowerNavigation';

export default () => 
  <div className="exam">
    <ExamUpperNavigation />
    <ExamBody />
    <ExamLowerNavigation />        
  </div>

