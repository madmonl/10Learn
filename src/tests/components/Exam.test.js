import React from 'react';
import { shallow } from 'enzyme';
import Exam from '../../components/Exam';

test('should render Exam correctly', () => {
  const wrapper = shallow(<Exam />);
  expect(wrapper).toMatchSnapshot();
});