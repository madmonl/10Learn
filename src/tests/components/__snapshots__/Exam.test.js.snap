// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Exam correctly 1`] = `
<div
  className="exam"
>
  <Connect(ExamUpperNavigation) />
  <Connect(ExamBody) />
  <WithStyles(Connect(ExamLowerNavigation)) />
</div>
`;
